### PR TITLE
fixed a few typos

### PR DESCRIPTION
**Pull request by e-marshall**
few things not included in this: 
- sst dataset link in computation notebook 
- in computation notebook it looks like the t90_fromt68() function is expected to return a np array to setup using apply_ufunc() in the next cell but it returns a xr.DataArray instead - not sure if intentional or not ?

### CODE REVIEW

File: fundamentals/03.1_computation_with_xarray.ipynb [hunk 1]
```diff
@@ -389,9 +391,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "mynewbook",
    "language": "python",
-   "name": "python3"
+   "name": "conda-env-mynewbook-py"
```
Comment (scottyhq) [inline fundamentals/03.1_computation_with_xarray.ipynb:396]:
```suggestion
   "name": "python3"
```

File: fundamentals/03.1_computation_with_xarray.ipynb [hunk 2]
```diff
@@ -389,9 +391,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "mynewbook",
```
Comment (scottyhq) [inline fundamentals/03.1_computation_with_xarray.ipynb:394]:
```suggestion
   "display_name": "Python 3 (ipykernel)",
```

**Comment by scottyhq** [comment]
> sst dataset link in computation notebook

It seems this might be an intermittent error, so let's leave for now (might need to change to https://psl.noaa.gov/thredds/dodsC/Datasets/noaa.ersst.v5/sst.mnmean.nc)...

> in computation notebook it looks like the t90_fromt68() 

Noted in https://github.com/xarray-contrib/xarray-tutorial/issues/61. I think we should remove the `gsw` section here and discuss `apply_ufunc` under Intermediate...


Thanks @e-marshall seems there is a kernel naming issue (maybe from nb_conda_kernels...) but otherwise happy to merge this and address these other issues later

**Review by scottyhq — APPROVED** [review]

**Comment by dcherian** [comment]
Nice work Emma!

**Pull request merged**